### ISSUE TITLE
Areas updated in 1.20 are not rendered anymore

### ISSUE THREAD

**Issue by alex4200** [overviewer/Minecraft-Overviewer#2092]
When I try to render areas that have been updated with version 1.20 they get not rendered anymore, i.e. it is just black. I tried to use the config with and without texturepath. For the texturepath I tried

`texturepath = "/Users/alex/Library/Application Support/minecraft/versions/1.20"`

and

`texturepath = "/Users/alex/Library/Application Support/minecraft/versions/1.20/1.20.jar"`

but neither seem to have any effect. How to fix this problem?

<img width="1285" alt="Screenshot 2023-06-09 at 10 14 05" src="https://github.com/overviewer/Minecraft-Overviewer/assets/12729291/3530ebae-4a08-49e3-9651-a357f4a9a971">

**Comment by serveradmin9912** [comment]
![image](https://github.com/overviewer/Minecraft-Overviewer/assets/81898871/eae58b24-38d9-41df-8988-310c1e433201)
![image](https://github.com/overviewer/Minecraft-Overviewer/assets/81898871/72c910bc-bc91-4d71-8f1d-dc30ec2174b7)

same here

**Comment by EuPhobos** [comment]
Yep, not working anymore.

**Comment by ajh123** [comment]
This wont be fixed look in README: `PLEASE NOTE: Overviewer is currently unmaintained. PRs will not be merged and issues will not be addressed. The website and repository will remain online and accessible`.

**Comment by ZarathonAndOn** [comment]
> This wont be fixed look in README: `PLEASE NOTE: Overviewer is currently unmaintained. PRs will not be merged and issues will not be addressed. The website and repository will remain online and accessible`.

That's a bummer.  Anyone got a good replacement?

I wonder if it's to do with this bug: https://bugs.mojang.com/browse/MC-263340

**Comment by Laephis** [comment]
BlueMap-Minecraft/BlueMap <https://github.com/BlueMap-Minecraft/BlueMap>

**Comment by ZarathonAndOn** [comment]
> BlueMap-Minecraft/BlueMap <https://github.com/BlueMap-Minecraft/BlueMap

Unfortunately, I'm running on Java Realms.  Had a simple download/render job that run 1/day.  BlueMap doesn't seem to support Vanilla or Realms, at a glance.

**Comment by CounterPillow** [comment]
You can run BlueMap standalone: https://bluemap.bluecolored.de/wiki/getting-started/Installation.html#using-bluemap-on-the-cli--standalone

**Comment by timawesomeness** [comment]
> I wonder if it's to do with this bug: https://bugs.mojang.com/browse/MC-263340

Doesn't appear to, that bug was fixed in 1.20.1 and this issue still occurs with 1.20.1.

**Comment by mtoensing** [comment]
Okay guys, I moved to BlueMap an will never look back: https://mc.marc.tv/3d/#overworld:-142:-6:-518:1500:0:0:0:0:perspective 

https://bluemap.bluecolored.de/ 

It is faster, it ins in 3d, the size of the rendered map is much smaller. And it is actively maintained.

**Comment by ZarathonAndOn** [comment]
Yeah, I will miss you overviewer.  Thank you for pointing me to bluemap.  I use realms and they have an offline/cli method that works fine.

**Comment by serveradmin9912** [comment]
I moved to BlueMap too: http://470m.de:8100/

**Comment by Gregory-AM** [comment]
I'm working on an updated version of Overviewer.
Supporting 1.20 worlds and textures.
https://github.com/GregoryAM-SP/The-Minecraft-Overviewer
